src/components/v2/OnlinePreviewPanel.vue

```diff
@@ -0,0 +1,195 @@
+const loadingPanel = ref<boolean>(true);
+const dependencies = ref<ThunderstoreMod[]>([]);
+
+function setActiveTab(tab: "README" | "CHANGELOG" | "Dependencies") {
+    activeTab.value = tab;
+}
+
+function fetchDataFor(mod: ThunderstoreMod, type: "readme" | "changelog"): Promise<string> {
+    return fetch(`https://thunderstore.io/api/experimental/package/${mod.getOwner()}/${mod.getName()}/${mod.getLatestVersion()}/${type}/`)
+        .then(res => res.json())
+        .then(res => res.markdown);
+}
+
+function fetchReadme(modToLoad: ThunderstoreMod) {
+    return fetchDataFor(props.mod, "readme").then(res => {
+        if (props.mod === modToLoad) {
+            readme.value = res;
+        }
+    });
+}
+
+function fetchChangelog(modToLoad: ThunderstoreMod) {
+    return fetchDataFor(props.mod, "changelog").then(res => {
+        if (props.mod === modToLoad) {
+            changelog.value = res;
+        }
+    });
+}
+
+function fetchAll(modToLoad: ThunderstoreMod) {
+    setActiveTab("README");
+    loadingPanel.value = true;
+    buildDependencies(modToLoad)
+        .then(value => dependencies.value = value)
+        .then(() => fetchReadme(modToLoad))
+        .then(() => fetchChangelog(modToLoad))
+        .then(() => {
+            if (props.mod === modToLoad) {
+                loadingPanel.value = false;
+            }
+        });
+}
+
+async function buildDependencies(mod: ThunderstoreMod) {
+    const builder: ThunderstoreCombo[] = [];
+    const tsVersion = await getVersionAsThunderstoreVersion(
+        store.state.activeGame.internalFolderName,
+        mod.getFullName(),
+        mod.getLatestVersion()
+    )
+    await ThunderstoreDownloaderProvider.instance.buildDependencySet(
+        tsVersion,
+        builder,
+        DependencySetBuilderMode.USE_LATEST_VERSION
+    );
+    return builder.map(value => value.getMod());
+}
+
+fetchAll(props.mod);
+watch(() => props.mod, (newValue) => {
+    fetchAll(newValue);
+});
+
+function getReadableDate(date: string): string {
+    return valueToReadableDate(new Date(date));
+}
+
+function getReadableCategories(mod: ThunderstoreMod) {
+    return mod.getCategories().join(", ") || "None";
+}
+
+const markdownToRender = computed(() => {
+    switch (activeTab.value) {
+        case "README": return readme.value;
+        case 'CHANGELOG': return changelog.value;
+        default: return null;
+    }
+});
+
+function showDownloadModal(mod: ThunderstoreMod) {
+    store.commit("openDownloadModModal", mod);
+}
+
+</script>
+
+<template>
+    <div class="c-preview-panel">
+        <div class="c-preview-panel__header">
+            <h1 class="title">{{ mod.getName() }}</h1>
+            <h2 class="subtitle">
+                By {{ mod.getOwner() }}
+            </h2>
+            <p class='card-timestamp'><strong>Downloads:</strong> {{mod.getDownloadCount()}}</p>
+            <p class='card-timestamp'><strong>Likes:</strong> {{mod.getRating()}}</p>
```

Comment:
Is the donation option intentionally omitted in this component? It's still available via the heart icon on the OnlineRowCard, but less prominent than it was in the old collapsible section.

Reply:
Looks like it got removed when I was playing around with the website link. Will re-add.

I do think it would make sense to move it in the future. It should probably belong in the installed list rather than the online section as I don't know why people would want to donate before having had downloaded the mod 🤔